src/System.Text.JsonLab.Dynamic/System.Text.JsonLab.Dynamic.csproj

```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="System.Dynamic.Runtime" Version="$(CoreFxStableVersion)" />
+    <PackageReference Include="System.Reflection.Emit.LightWeight" Version="$(CoreFxStableVersion)" />
```

Comment:
You said you don't yet do ref.emit.

Reply:
I am starting to play with ref.emit (the instance of T returned is constructed using ref emit instead of Activator.CreateInstance). What I meant was that the property setters are still using reflection.